xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/BlogParameters.xml

```diff
@@ -74,6 +74,6 @@
  * @param $imgName The name of the icon to use (without the extension).
  * @param alt The alternate text that should be displayed when images are not viewable.
  *#
-#macro(toolImage $imgName $alt)[[image:path:$xwiki.getSkinFile("icons/silk/${imgName}.png")||class="icon-manage" alt="${alt}"]]#end
+#macro(toolImage $imgName $alt)(% class="glyphicon glyphicon-${imgName} icon-manage" spam="${alt}"%) #end
```

Comment:
Hello, What do you mean? I can  not use `<img>` because glyphicon must use `<span>`:
http://www.w3schools.com/bootstrap/bootstrap_ref_comp_glyphs.asp
(btw edit menu (id="tmEdit") use span too to display icons).
Maybe you want something like this?
`[[(% class="glyphicon glyphicon-globe icon-manage" spam="${alt}"%) >>path:$doc.getURL('delete')]]`
or replace #macro(toolImage by #macro(toolGlyphicon ?